src/components/progress-steps/progress-steps.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { cn } from '@/utilities/functions';
+import { Plus, Check } from 'lucide-react';
+import { getVariantClasses, completedStepCommonClasses, stepWrapperClasses } from './utils';
+
+const ProgressSteps = ( {
+	variant = 'dot',
+	size = 'sm',
+	type = 'inline',
+	currentStep = 1,
+	children,
+	className,
+	...rest
+} ) => {
+	const totalSteps = React.Children.count( children );
+
+	const sizeClasses = {
+		sm: {
+			dot: 'size-2.5',
+			ring: 'size-5',
+			numberIcon: 'size-5 text-[10px]',
```

Comment:
Add this `text-[10px]` in the Tailwind CSS configuration and use the `rem` value.

Reply:
done